### DISCUSSION TITLE
Unable to get account config error

### DISCUSSION THREAD

**Discussion by kglaynyi** [masterking32/MasterHamsterKombatBot#114] [Q&A] [answered]
It's all good for earlier.After I stop and then start again later, it show like "Unable to get account config error". Any Ideas?

**Comment by msunaramadhanilmii** [comment]
I also get the same error, waiting for an answer.

**Reply by epis78** [reply]
me 2

**Comment by masterking32** [comment, marked as answer]
Fixed:
https://github.com/masterking32/MasterHamsterKombatBot/pull/115
https://github.com/masterking32/MasterHamsterKombatBot/issues/113
https://github.com/masterking32/MasterHamsterKombatBot/commit/c681ccfac9cddc5a6dbc365fd3a8e6f761ce00b6
